index.html

```diff
@@ -6,216 +6,266 @@
     <title>My Portfolio</title>
 </head>
 
 <body>
     <header>
         <nav>
             <span class="logo">Welcome</span>
-            <img src="./img/Union.svg" alt="Hamburger menu icon" />
+            <img class="burger-menu"src="./img/Union.svg" alt="Hamburger menu icon" />
         </nav>
     </header>
 
+    <nav class="head">
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+
     <section class="headline">
         <div class="headline-content">
-            <h1 class="headline-title">Hey there. I'm Patrick<br>
+            <h1 class="headline-title">
+                Hey there. I'm Patrick<br />
                 <span style="color: #36b37e">I'm a software developer</span>
             </h1>
-            <p>
-                I can help you build a product , feature or website Look through some of
-                my work and experience! If you like what you see and have a project you
-                need coded, don't hestiate to contact me.
+            <p class="change">
+                I can help you build a product , feature or website Look through some
+                of my work and  experience! If you like what you see and have a project
+                you need coded, don't hestiate to contact me.
             </p>
 
             <ul class="social-media">
                 <li>
-                    <a href=""><img src="./img/twitter.svg" alt="Twitter icon"></a>
+                    <a href=""><img src="./img/twitter.svg" alt="Twitter icon" /></a>
                 </li>
                 <li>
-                    <a href=""><img src="./img/Linkedin icon.svg" alt="Linkedin icon"></a>
+                    <a href=""><img src="./img/Linkedin icon.svg" alt="Linkedin icon" /></a>
                 </li>
                 <li>
-                    <a href=""><img src="./img/medium.svg" alt="Medium icon"></a>
+                    <a href=""><img src="./img/medium.svg" alt="Medium icon" /></a>
                 </li>
                 <li>
-                    <a href=""><img src="./img/github.svg" alt="Github icon"></a>
+                    <a href=""><img src="./img/github.svg" alt="Github icon" /></a>
                 </li>
                 <li>
-                    <a href=""><img src="./img/angelist.svg" alt="Angelist icon"></a>
+                    <a href=""><img src="./img/angelist.svg" alt="Angelist icon" /></a>
                 </li>
             </ul>
         </div>
     </section>
 
     <div class="disabled">
         <button type="button">
-            <img src="./img/chevron-down.svg" alt="disabled button">
+            <img src="./img/chevron-down.svg" alt="disabled button" />
         </button>
     </div>
 
-
     <section>
-        <div class="headline-title" style="padding: 0 30px;">
-            My Recent Works
+        <div class="headline-title" style="padding: 0 30px">
+            <h2>My Recent Works</h2>
```

Comment:
- [ ] Its better if you use external styling instead of inline styling. you can give it a class and targeted on a separate CSS file. its HTML and CSS best practice.